docs/services/data/_index.md

```diff
@@ -1,147 +1,742 @@
+| [Arm](/components/arm/)                         | `EndPosition`, `JointPositions` |
+| [Board](/components/board/)                     | `Analogs`, `Gpios` |
+| [Camera](/components/camera/)                   | `GetImages`, `ReadImage`, `NextPointCloud` |
+| [Encoder](/components/encoder/)                 | `TicksCount` |
+| [Gantry](/components/gantry/)                   | `Lengths`, `Position` |
+| [Motor](/components/motor/)                     | `Position`, `IsPowered` |
+| [Movement sensor](/components/movement-sensor/) | `AngularVelocity`, `CompassHeading`, `LinearAcceleration`, `LinearVelocity`, `Orientation`, `Position` |
+| [Sensor](/components/sensor/)                   | `Readings` |
+| [Servo](/components/servo/)                     | `Position` |
+| [Vision service](/services/vision/)             | `CaptureAllFromCamera` |
+
+{{% /tab %}}
+{{% tab name="viam-micro-server" %}}
+
+<!-- prettier-ignore -->
+| Type | Method |
+| ---- | ------ |
+| [Movement Sensor](/components/movement-sensor/) | [`AngularVelocity`](/appendix/apis/components/movement-sensor/#getangularvelocity), [`LinearAcceleration`](/appendix/apis/components/movement-sensor/#getlinearacceleration), [`LinearVelocity`](/appendix/apis/components/movement-sensor/#getlinearvelocity) |
+| [Sensor](/components/sensor/) | [`GetReadings`](/appendix/apis/components/sensor/#getreadings) |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## View captured data
+
+To view all the captured data you have access to, go to the [**DATA** tab](https://app.viam.com/data/view) where you can filter by location, type of data, and more.
+
+You can also access data from a resource, machine part, or machine menu.
+
+## Considerations
+
+- **Capturing too much data**: You can [use filtering to collect and sync only certain images](/how-tos/image-data/#use-filtering-to-collect-and-sync-only-certain-images) to capture data selectively.
+- **Rentention policy**: Set a `retention_policy` attribute in your [data capture configuration](#resource-data-capture-configuration) to avoid keeping data stored in the Viam cloud longer than a specified number of days.
+- **Pausing sync**: You can pause cloud sync at any time by navigating to your machine's **CONFIGURE** tab and disabling **Syncing** for your [data management service](../).
+
+  If you have captured data that you do not want to sync, delete the data on the machine before resuming cloud sync.
+  To delete the data locally, `ssh` into your machine and delete the data in the directory where you capture data.
+
+- **Sync data conditionally**: You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+  For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+  To set up triggers for syncing see:
```

Comment:
was this supposed to say something else or indeed supposed to lead to API? just checking